indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
i wonder if sass is smart enough to strip all but one of these comments. otherwise we add tons of noise when keeping all license headers in the generated css file